examples/landmarks/resources.html

```diff
@@ -54,7 +54,7 @@ <h2>Read More</h2>
                             <li><a href="https://www.w3.org/TR/WCAG20-TECHS/aria#ARIA20">ARIA20: Using the region role to identify a region of the page</a> (WCAG 2.0 Technique)</li>
                             <li><a href="https://alistapart.com/column/wai-finding-with-aria-landmark-roles">WAI-finding with ARIA Landmark Roles</a> (A List Apart)</li>
                             <li><a href="https://www.paciellogroup.com/blog/2013/02/using-wai-aria-landmarks-2013/">Using WAI-ARIA Landmarks – 2013</a> (Paciello Group)</li>
-                            <li><a href="https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks.html">HTML 5 and ARIA Landmarks</a> (Deque University)</li>
+                            <li><a href="https://dequeuniversity.com/assets/html/jquery-summit/html5/slides/landmarks.html">HTML and ARIA Landmarks</a> (Deque University)</li>
```

Comment:
Might want to leave the 5 in this link text because it is the name of the linked document.

Reply:
Reverted